raiden/network/pathfinding.py

```diff
@@ -281,7 +281,7 @@ def configure_pfs_or_exit(
         )
         if maybe_pfs_url is None:
             raise RaidenError(
-                "The Service Registry has no registered Pathfinding Service "
+                "No registered Pathfinding service seems to be running"
```

Comment:
```suggestion
                "No registered Pathfinding Service seems to be running"
```

Reply:
also a good point. This seems not to be consistent here. We use is lower case in the rest of the messages

Reply:
Alright. As long as we are consistent I am happy. I just saw that it used to be **S**ervice in the previous message, so wanted to stick with it. But if it's lower case all other place, we should stick to that.